sympy/physics/vector/frame.py

```diff
@@ -720,57 +720,38 @@ def orient_explicit(self, parent, dcm):
         dcm : Matrix, shape(3, 3)
             Direction cosine matrix that specifies the relative rotation
             between the two reference frames.
-
+        rotation matrix : Matrix Rotation Direction
+            The simple matrix rotation relative ``parent`` to ``dcm``.
         Warns
         ======
 
         UserWarning
             If the orientation creates a kinematic loop.
 
-        Examples
-        ========
-
-        Setup variables for the examples:
-
-        >>> from sympy import symbols, Matrix, sin, cos
-        >>> from sympy.physics.vector import ReferenceFrame
-        >>> q1 = symbols('q1')
-        >>> A = ReferenceFrame('A')
-        >>> B = ReferenceFrame('B')
-        >>> N = ReferenceFrame('N')
-
-        A simple rotation of ``A`` relative to ``N`` about ``N.x`` is defined
-        by the following direction cosine matrix:
+        """
+        self.orient_dcm(parent = parent, dcm = dcm)
 
-        >>> dcm = Matrix([[1, 0, 0],
-        ...               [0, cos(q1), -sin(q1)],
-        ...               [0, sin(q1), cos(q1)]])
-        >>> A.orient_explicit(N, dcm)
-        >>> A.dcm(N)
-        Matrix([
-        [1,       0,      0],
-        [0,  cos(q1), sin(q1)],
-        [0, -sin(q1), cos(q1)]])
+    def orient_dcm(self, parent, dcm):
+        """Sets the orientation of this reference frame relative to a parent
+        reference frame by explicitly setting the direction cosine matrix.
 
-        This is equivalent to using ``orient_axis()``:
+        Parameters
+        ==========
 
-        >>> B.orient_axis(N, N.x, q1)
-        >>> B.dcm(N)
-        Matrix([
-        [1,       0,      0],
-        [0,  cos(q1), sin(q1)],
-        [0, -sin(q1), cos(q1)]])
+        parent : ReferenceFrame
+            Reference frame that this reference frame will be rotated relative
+            to.
+        dcm : Matrix, shape(3, 3)
+            Direction cosine matrix that specifies the relative rotation
+            between the two reference frames.
+        rotation matrix : Matrix Rotation Direction
+            The simple matrix rotation relative ``parent`` to ``dcm``.
```

Comment:
The same change is required here. Only now the DCM describes the orientation of the child with respect to the parent.